lib/events.js

```diff
@@ -256,9 +256,14 @@ function _addListener(target, type, listener, prepend) {
       m = $getMaxListeners(target);
       if (m && m > 0 && existing.length > m) {
         existing.warned = true;
-        process.emitWarning('Possible EventEmitter memory leak detected. ' +
+        const w = new Error('Possible EventEmitter memory leak detected. ' +
                             `${existing.length} ${type} listeners added. ` +
                             'Use emitter.setMaxListeners() to increase limit');
+        w.name = 'Warning';
```

Comment:
Yeah, good point. Nevermind then

On Saturday, August 27, 2016, Anna Henningsen <notifications@github.com>
wrote:

> In lib/events.js
> <https://github.com/nodejs/node/pull/8298#discussion_r76515949>:
>
> >                              `${existing.length} ${type} listeners added. ` +
> >                              'Use emitter.setMaxListeners() to increase limit');
> > +        w.name = 'Warning';
>
> +1 … but that would be a semver-major change?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/nodejs/node/pull/8298/files/802e6be4c7e2bfd08224fb6769fbe7b2d8ed0239#r76515949>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAa2eW6tZ7CDeSmG54J9RAysM9fS9TBZks5qkDgHgaJpZM4Jupwl>
> .
>